.github/workflows/stubsabot.yml

```diff
@@ -17,6 +17,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          ssh-key: ${{ secrets.STUBSABOT_SSH_PRIVATE_KEY }}
```

Comment:
```suggestion
          # use an ssh key so that checks automatically run on stubsabot PRs
          ssh-key: ${{ secrets.STUBSABOT_SSH_PRIVATE_KEY }}
```